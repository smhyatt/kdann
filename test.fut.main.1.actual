[18.172409f64, 1.020947f64, 17.590776f64, 1.666701f64, 5.105632f64, 2.312449f64,
 0.0f64]
