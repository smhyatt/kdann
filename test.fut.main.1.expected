[42.10298793f64, -79.7838349f64, 2.60175886f64, 15.31100977f64, -1.51816989f64,
 0.33231584f64, -0.32382814f64, 0.74201471f64]
